pytorch_pfn_extras/training/manager.py

```diff
@@ -435,6 +436,14 @@ def get_extension(self, name: str) -> extension_module.Extension:
         else:
             raise ValueError('extension %s not found' % name)
 
+    def _run_on_error(self, exc: Exception) -> None:
+        if not self._run_on_error_called:
+            self._run_on_error_called = True
+            tb = exc.__traceback__
+            assert tb is not None
+            for _, entry in self.extensions:
+                entry.extension.on_error(self, exc, tb)
```

Comment:
I think we need to check if the extension has `on_error` method.

Reply:
discussed offline, `entry` is always wrapped so `on_error` is guaranteed to be present.